frontend/src/assets/styles/theme.ts

```diff
@@ -39,4 +43,25 @@ const THEME_DARK = {
     media_xxl: "1536px"
 };
 
-export { THEME_DARK };
+const scrollBar = css`
```

Comment:
Move it to theme because you don't need to import that variable then. 
Use `${({ theme }) => theme.searchBar}`